packages/ra-supabase-core/src/useRedirectIfAuthenticated.ts

```diff
@@ -0,0 +1,26 @@
+import { useCheckAuth } from 'ra-core';
+import { useEffect } from 'react';
+import { useHistory } from 'react-router';
+
+/**
+ * This hook redirect the user to the provided path (/ by default) if they are authenticated.
+ **/
+export const useRedirectIfAuthenticated = (
```

Comment:
Great hook, I think we should definitely have it in the react-admin core

Reply:
Will backport when I'll have time